cime_config/acme/machines/Makefile

```diff
@@ -185,14 +185,41 @@ ifndef AR
    AR := ar
 endif
 
-ifdef NETCDF_PATH
-  ifndef INC_NETCDF
-    INC_NETCDF:=$(NETCDF_PATH)/include
-  endif
-  ifndef LIB_NETCDF
-    LIB_NETCDF:=$(NETCDF_PATH)/lib
+# To prevent breaking cleaning and building components which don't require NetCDF directly
+# NetCDF is only required for building the coupled model and the PIO library
+ifeq ($(if $(or $(findstring PIO, $(MODEL)), $(findstring CIME_MODEL, $(MODEL))), true, false), true)
```

Comment:
CIME_MODEL is not going to be found in $(MODEL) cesm or acme is.  cesm in the cesm Makefile, acme in the acme one.